### PR TITLE
use the lowercased spaceId as the key phrase for deriving the key and iv

### PR DESCRIPTION
**Pull request by tak-hntlabs**

### CODE REVIEW

File: packages/sdk/src/client.ts [hunk 1]
```diff
@@ -869,7 +869,8 @@ export class Client
         })
 
         // encrypt the chunked media
-        const { key, iv } = await deriveKeyAndIV(spaceId)
+        // use the lowercased spaceId as the key phrase
+        const { key, iv } = await deriveKeyAndIV(spaceId.toLowerCase())
```
[the remark this answers is not in the document]
Comment (tak-hntlabs) [reply packages/sdk/src/client.ts]: good point. Fixed

**Review by texuf — APPROVED** [review]
Thanks this tiny change makes a huge difference.

File: packages/sdk/src/client.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (texuf) [reply packages/sdk/src/client.ts]: Why did you need to lowercase it?

[the remark this answers is not in the document]
Comment (tak-hntlabs) [reply packages/sdk/src/client.ts]: ran into this problem when testing with the fetch-stream service. If someone use an Ethereum checksum address in the URL, the fetch-stream service isn't able to decrypt the EncryptedData in the spaceImage to see the actual media stream ID. 
It was simpler to just make the context always use all lowercase.

**Pull request merged**